src/SignalR/clients/ts/signalr/src/HubConnection.ts

```diff
@@ -61,11 +75,11 @@ export class HubConnection {
     // create method that can be used by HubConnectionBuilder. An "internal" constructor would just
     // be stripped away and the '.d.ts' file would have no constructor, which is interpreted as a
     // public parameter-less constructor.
-    public static create(connection: IConnection, logger: ILogger, protocol: IHubProtocol): HubConnection {
-        return new HubConnection(connection, logger, protocol);
+    public static create(connection: IConnection, logger: ILogger, protocol: IHubProtocol, reconnectPolicy?: IReconnectPolicy): HubConnection {
```

Comment:
Should we just add this as an additional overload instead of the breaking change

Reply:
It's private to TypeScript. If called from JavaScript or without type checks without the last argument reconnectPolicy is just undefined I think. Same goes for the HubConnection ctor.

That's part of reason I made reconnectPolicy optional instead of creating a NullReconnectPolicy or similar.